test/route-define-test.js

```diff
@@ -441,6 +455,10 @@ QUnit.test("triggers __url event anytime a there's a change to individual proper
 	var done = assert.async();
 	canRoute.start();
 
+	var timeoutID = setTimeout(function page_two() {
+		canRoute.data.page = "two";
+	}, 50);
+
```

Comment:
Some other tests failed because the `data` would still be set after this test runs. This prevents that.